### PR TITLE
Add dependency on contract-external-libraries-go

### PR DESCRIPTION
**Pull request by netoneko**
Following conversation with @OdedWx and @erankirsh:
- [x] struct serialization extracted to a separate repo: https://github.com/orbs-network/contract-external-libraries-go
- [x] added this repo as a dependency for **experimental** builds

### CODE REVIEW

File: docker/build/Dockerfile.export [hunk 1]
```diff
@@ -3,6 +3,7 @@ FROM golang:1.11.4-alpine
 RUN apk add --no-cache gcc musl-dev
 
 ADD ./vendor/github.com/orbs-network/orbs-contract-sdk/go/ /go/src/github.com/orbs-network/orbs-network-go/vendor/github.com/orbs-network/orbs-contract-sdk/go/
+ADD ./vendor/github.com/orbs-network/contract-external-libraries-go/ /go/src/github.com/orbs-network/orbs-network-go/vendor/github.com/orbs-network/contract-external-libraries-go/
```
Comment (electricmonk) [inline docker/build/Dockerfile.export]: should this be here AND in the experimental file?

File: docker/build/Dockerfile.gamma [hunk 2]
```diff
@@ -3,6 +3,7 @@ FROM golang:1.11.4-alpine
 RUN apk add --no-cache gcc musl-dev
 
 ADD ./vendor/github.com/orbs-network/orbs-contract-sdk/go/ /go/src/github.com/orbs-network/orbs-network-go/vendor/github.com/orbs-network/orbs-contract-sdk/go/
+ADD ./vendor/github.com/orbs-network/contract-external-libraries-go/ /go/src/github.com/orbs-network/orbs-network-go/vendor/github.com/orbs-network/contract-external-libraries-go/
```
Comment (electricmonk) [inline docker/build/Dockerfile.gamma]: ditto

File: docker/build/Dockerfile.export (hunk 1 above)
Comment (netoneko) [reply docker/build/Dockerfile.export]: Fixed.

**Comment by OdedWx** [comment]
Looks good, 
Note that ClearStruct is equivalent to WriteStruct with an empty struct as provided to ClearStruct. Worth mentioning in MD.

**Comment by erankirsh** [comment]
I hereby consent to adding the external libraries.
Note that Orbs Production network will NOT include these external libraries and it should only be used in Gamma for development reference. 
Once a library has been deemed necessary - Orbs network will implement it in the SDK to integrate it into production network.

**Pull request merged**